coalib/output/ConsoleInteraction.py

```diff
@@ -566,21 +566,31 @@ def choose_action(console_printer, actions):
     :return:                Return choice of action of user.
     """
     while True:
+        possible_actions = []
         console_printer.print(format_lines('(D)o nothing'))
+        possible_actions.append('D')
         for i, action in enumerate(actions, 1):
-            console_printer.print(format_lines('{:>2}: {}'.format(
-                i,
+            console_printer.print(format_lines('{}'.format(
                 action.desc)))
-
+            if action.desc == '(O)pen file':
```

Comment:
also, wouldnt it be better to use a dict? such as:
dict['O'] = false // if not applicable
dict['O'] = true // if applicable
and youd just make them all false at the start of the loop, and instead of inserting, youd just set them true :) hows that sound?